app/parse.py

```
@@ -8,8 +17,61 @@ class Quote:
     tags: list[str]
 
 
+QUOTE_FIELDS = [field.name for field in fields(Quote)]
+
+
+def get_page(url: str) -> bytes:
+    for attempt in range(3):
```

Nice: a retry loop for network fetching (3 attempts) is implemented, and the function raises an error if all attempts fail. This is safer than silently returning empty content and aligns with a robust fetching strategy.